cmd/clusterctl/client/cluster/cert_manager.go

```diff
@@ -401,7 +402,9 @@ func (cm *certManagerClient) getManifestObjs(certManagerConfig config.CertManage
 	// Given that cert manager components yaml are stored in a repository like providers components yaml,
 	// we are using the same machinery to retrieve the file by using a fake provider object using
 	// the cert manager repository url.
-	certManagerFakeProvider := config.NewProvider("cert-manager", certManagerConfig.URL(), "")
+	// We replace "latest" in the url by the wanted version to disable auto version-discovery in repositoryClientFactory.
```

Comment:
what about
```suggestion
	// In order to avoid unnecessary calls to GitHub API, "latest" is replace with the version currently installed by clusterctl.
```